scripts/run_quality_checks.py

```
@@ -0,0 +1,75 @@
+"""
+
+import subprocess
+import sys
+from pathlib import Path
+from typing import List, Tuple
+
+def run_command(cmd: List[str], description: str) -> Tuple[bool, str]:
+    """Run a command and return success status and output."""
+    print(f"Running {description}...")
+    try:
+        result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)